api/src/main/java/io/opentelemetry/distributedcontext/DistributedContext.java

```diff
@@ -74,9 +74,7 @@
     Builder setParent(DistributedContext parent);
 
     /**
-     * Sets the option to become a {@link DistributedContext} with no parent. If not set, the value
-     * of {@link DistributedContextManager#getCurrentContext()} at {@link #build()} time will be
```

Comment:
I think this statement still applies. If `setNoParent` is never called, implicit parent will still be used, while if `setNoParent` is called, neither implicit nor explicit parent will be used.

Reply:
I'll rephrase it to clarify it a bit better.